MWSE/TES3Vectors.cpp

```diff
@@ -68,6 +68,25 @@ namespace TES3 {
 		return str;
 	}
 
+	float Vector2::distance(const Vector2* vec2) const {
+		float dx = x - vec2->x;
+		float dy = y - vec2->y;
+		return sqrt(dx * dx + dy * dy);
+	}
+
+	
+	float Vector2::distanceChebyshev(const Vector2* vec2) const {
+		float dx = fabs(x - vec2->x);
+		float dy = fabs(y - vec2->y);
+		return std::max(dx,dy);
```

Comment:
Can you insert space after a comma in argument lists to stay consistent with the rest of the code?